### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by karanpreetkaur**
Adding CONTRIBUTING.md 

Note- The link to Code of Conduct will be added once we have that file in master repository.

### CODE REVIEW

**Review by thayeylolu — APPROVED** [review]
looks good to me!

File: CONTRIBUTING.md
Comment (liannah) [inline CONTRIBUTING.md:14]: do we have such files ?

Comment (Arushi282) [reply CONTRIBUTING.md]: yeah I think since we are working with python files, you can mention ".py" and docstrings instead?

Comment (karanpreetkaur) [reply CONTRIBUTING.md]: @liannah Thanks for highlighting it , I have updated it now. So, basically the idea here is to allow contributors to make docstring or comments changes directly but to restrict editing the source codes without creating issue

Comment (karanpreetkaur) [reply CONTRIBUTING.md]: @Arushi282 Can you do review now ?

Comment (liannah) [reply CONTRIBUTING.md]: Looks good to me, I am approving. We can wait for @Arushi282 's approval as well.

**Review by liannah — APPROVED** [review]
Looks good to me

File: CONTRIBUTING.md
Comment (Arushi282) [reply CONTRIBUTING.md]: Hi @karanpreetkaur looks great, thanks for making the changes, and the contributing file is basically how others who are not in the core group can make changes to the project on github. Will merge this now.

**Pull request merged**
